lib/jekyll/drops/url_drop.rb

```diff
@@ -35,46 +35,87 @@ def categories
         category_set.to_a.join("/")
       end
 
+      # CCYY
       def year
         @obj.date.strftime("%Y")
       end
 
+      # MM: 01..12
       def month
         @obj.date.strftime("%m")
       end
 
+      # DD: 01..31
       def day
         @obj.date.strftime("%d")
       end
 
+      # hh: 00..23
       def hour
         @obj.date.strftime("%H")
       end
 
+      # mm: 00..59
       def minute
         @obj.date.strftime("%M")
       end
 
+      # ss: 00..59
       def second
         @obj.date.strftime("%S")
       end
 
+      # D: 1..31
       def i_day
         @obj.date.strftime("%-d")
       end
 
+      # M: 1..12
       def i_month
         @obj.date.strftime("%-m")
       end
 
+      # MMM, uppercase: JAN..DEC
       def short_month
-        @obj.date.strftime("%b")
+        @obj.date.strftime("%^b") 
```

Comment:
Why is this being *changed* to UPCASE?
How would one get `Jan..Dec` now?

Reply:
I actually do not remember why. Will change it back.